content/docs/user-guide/networking/multiplayer/_index.md

```diff
@@ -26,7 +26,6 @@ The Multiplayer Gem supports the following:
 | [Configuring a Project](configuration) | How to add and enable the O3DE Multiplayer Gem in a project. |
 | [Running Multiplayer Projects](running) | How to run projects that use the O3DE Multiplayer Gem. |
 | [Multiplayer Auto-components](autocomponents) | How to automatically create components for use with the Multiplayer Gem using the AzAutoGen system. |
-| [Separating Client and Server](code_separation) | How to separate client and server logic and build dependencies.
```

Comment:
This should be retained, want client/server split docs to part of published docs.